resources/definitions/fdmprinter.def.json

```diff
@@ -4622,11 +4622,12 @@
                     "label": "Enable Prime Blob",
                     "description": "Whether to prime the filament with a blob before printing. Turning this setting on will ensure that the extruder will have material ready at the nozzle before printing. Printing Brim or Skirt can act like priming too, in which case turning this setting off saves some time.",
                     "type": "bool",
-                    "resolve": "any(extruderValues('prime_blob_enable'))",
+                    "resolve": "any(extruderValues('prime_blob_enable')) and resolveOrValue('print_sequence') != 'one_at_a_time'",
                     "default_value": false,
                     "settable_per_mesh": false,
                     "settable_per_extruder": true,
-                    "enabled": false
+                    "enabled": false,
+                    "warning_value": "True if resolveOrValue('print_sequence') == 'one_at_a_time' else None"
```

Comment:
Is there a reason why this should evaluate to `None` when there needs to be no warning, rather than to `False`? If it can evaluate to `False` we can simplify this to:

    "warning_value": "resolveOrValue('print_sequence') == 'one_at_a_time'"

Reply:
I tried with the simplified formula and the behaviour seemed to be the same. I think we can simplify this.

Reply:
I think it's _meant_ to represent a three-state value though: a. warn on true/enabled, b. warn on false/disabled and c. don't warn. If the behaviour doesn't change to reflect that, it proably explains the 'other' :-) bug that you found.

Reply:
Ah, yes that makes sense.